pg_search/src/index/search.rs

```diff
@@ -141,6 +139,12 @@ impl SearchIndex {
         SEARCH_INDEX_MEMORY.insert(self.directory.clone(), self);
     }
 
+    pub unsafe fn get_writer() -> &'static mut Writer {
```

Comment:
I mention this SAFETY note here because it's true and we need to make sure our code isn't calling "create index" (or whatever adds a new entry to SEARCH_INDEX_MEMORY) while we have a borrow from this function.  If so, that borrow becomes invalid and it's instant crash.